### DISCUSSION TITLE
Email Example from Website

### DISCUSSION THREAD

**Discussion by ivanjeremic** [shadcn-ui/ui#2398] [General]
Is the Email Example from https://ui.shadcn.com/examples/mail available as code somewhere in the repo?

**Comment by adebiyial** [comment]
Here: https://github.com/shadcn-ui/ui/tree/main/apps/www/app/examples/mail

**Comment by nghiatv** [comment]
https://github.com/shadcn-ui/ui/tree/main/apps/www/app/(app)/examples/mail updated

**Comment by ksvighnesh** [comment]
This sidebar is uniquely different from the given sidebar block components  👀
